app/views/hub/coalitions/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @title = t(".title") %>
```

Comment:
Trivial nit: This `@title = ...` pattern is a little weird. I know it's copy-pasta, but if this view works fine without modifying instance variables, it would make 1% happier.